multiqc/utils/search_patterns.yaml

```diff
@@ -20,6 +20,20 @@ bamtools/stats:
   contents: "Stats for BAM file(s):"
   shared: true
   num_lines: 10
+bases2fastq/persample:
+  fn: "*_stats.json"
+  contents_re: ".*NumPolonies.*"
+  num_lines: 100
+bases2fastq/run:
+  fn: "RunStats.json"
+  contents_re: ".*NumPolonies.*"
+  num_lines: 100
+bases2fastq/project:
+  fn: "*_RunStats.json"
+  contents_re: ".*NumPolonies.*"
+  num_lines: 100
+bases2fastq/group:
+  fn: "*_b2fgroup.csv"
```

Comment:
We don't seem to have any test data for the group file, do we? 
https://github.com/ewels/MultiQC_TestData/tree/master/data/modules/bases2fastq

Would it be possible to add some? @blajoie